tv_series.md

```diff
@@ -81,44 +93,65 @@
 > Set in the 1980s, this series dramatizes the personal computing boom through the eyes of a visionary, an engineer and a prodigy whose innovations directly confront the corporate behemoths of the time. Their personal and professional partnership will be challenged by greed and ego while charting the changing culture in Texas' Silicon Prairie.
 
 ![halt_and_catch_fire](assets/halt_and_catch_fire.jpg)
-* Year: 2014-2017
-* Country: USA
-* [IMDb](https://www.imdb.com/title/tt2543312/)
+
+- Year: 2014-2017
+- Country: USA
+- [IMDb](https://www.imdb.com/title/tt2543312/)
 
 ### 8.2
 
 #### Chuck
+
```

Comment:
here.. and another places below too

Reply:
Yes, my bad for that my code editor had prettier on and it affected the spacing.

Reply:
Could you please fix it?

Reply:
I am closing this then and redo it